perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ModulesUtilsBlImpl.java

```diff
@@ -481,14 +486,40 @@ public boolean haveRightToWriteAttributeInAnyGroupOrResource(PerunSessionImpl se
 		return groupNameNamespaces;
 	}
 
-	public void checkReservedUnixGroupNames(Attribute groupNameAttribute) throws WrongAttributeValueException {
+	public void checkReservedUnixGroupNames(Attribute groupNameAttribute) throws InternalErrorException, WrongAttributeValueException {
 		if(groupNameAttribute == null) return;
-		if(reservedNamesForUnixGroups.contains(groupNameAttribute.getValue())) throw new WrongAttributeValueException(groupNameAttribute, "This groupName is reserved.");
+
+		if (perunNamespaces == null) {
+			perunNamespaces = BeansUtils.getAllPropertiesFromCustomConfiguration("perun-namespaces.properties");
```

Comment:
If file not found exception is thrown (when file not exists, correct status), Internal Error Exception is thrown and everything fail. In this case we need to set perunNamespaces map to empty one and continue.